packages/plugin-infra/src/web3-types.ts

```diff
@@ -21,6 +21,10 @@ export enum TokenType {
     NonFungible = 'NonFungible',
 }
 
+type ColorRgb = `rgb(${number}, ${number}, ${number})`
+type ColorHex = `#${number}${number}${number}${number}${number}${number}` | `#${number}${number}${number}`
+type Color = ColorRgb | ColorHex
```

Comment:
Might refactor as utility types

Reply:
missing RGBA pattern

<img width="600" alt="CleanShot 2022-01-13 at 14 50 03@2x" src="https://user-images.githubusercontent.com/3842474/149279807-c93ed15c-05c9-4d84-9dba-fd6a16ef4127.png">

Reply:
Add `rgba` and `hsl`